features/ingress-egress-gateway.md

```diff
@@ -0,0 +1,224 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+Gateways: Ingress, Egress for all protocols
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary lead(s):**
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+**Short description:**
+
+Support Ingress and Egress traffic for all protocols
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
+
+
+**Design Docs:**
+[BTS for gateway](https://docs.google.com/document/d/1DxwqhNqxm1r3fSomyQgJWy103bv29B0ZTEvu_hZqQgc/edit?resourcekey=0-uPN2Gtx-BHvj-sAvN9SGKA#)
```

Comment:
It feels odd to have an un-implemented future looking doc as the core design doc for a feature implemented + stable over 3 years ago.

Reply:
Removed it. Thanks John